monitoring/metrics/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Metrics
+summary: 
+reviewed: 2018-01-12
+component: Metrics
+versions: 'Metrics:*'
+---
+
+The Metrics plugin collects metric data about the performance of running endpoints. This data can be forwarded to a ServiceControl Monitoring instance and then viewed in ServicePulse.
+
+To see performance monitoring in action, try our [standalone demo](/tutorials/monitoring-demo/).
+
+For a full list of the performance metrics captured and their formal definitions, see [Metric definitions](definitions.md).
+
+```mermaid
+graph LR
```

Comment:
![image](https://user-images.githubusercontent.com/152998/34988767-2cfb6ffa-fac0-11e7-8ba0-3f77397f00d5.png)


Here too, the arrow between SP and SC seems wrong as SP is querying the data.

Reply:
I mean, the line between SC Monitoring and the Metrics queue is also around the wrong way. This is showing data flow rather than component connections. Is that the more valuable thing to show? Should we label it?